packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -315,13 +336,22 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
                         )
                       : this.renderTriggerButton(styles, rtl, getToggleButtonProps)}
                   </div>
-                  {Indicator.create(toggleIndicator, {
-                    defaultProps: {
-                      direction: isOpen ? 'top' : 'bottom',
-                      onClick: getToggleButtonProps().onClick,
-                      styles: styles.toggleIndicator,
-                    },
-                  })}
+                  {showClearIndicator
+                    ? Icon.create(clearIndicator, {
+                        defaultProps: {
+                          className: Dropdown.slotClassNames.clearIndicator,
+                          onClick: this.handleClear,
```

Comment:
the search input/ trigger button is not focused after clicking on the clear button; see:

![screen recording 2019-02-12 at 14 24 11](https://user-images.githubusercontent.com/5442794/52638351-16151100-2ed2-11e9-9ea7-ae6a84519738.gif)

you need to call

```typescript
this.tryFocusSearchInput()
this.tryFocusTriggerButton()
```

in the click handler or maybe write a new function

```typescript
private tryFocusTrigger = () => {
  if (this.props.search) {
    this.inputRef.current.focus()
  } else {
    this.buttonRef.current.focus()
  }
}
```

to incorporate the logic of what trigger needs to be focused depending on props

Reply:
Nice catch 👍  Reused existing methods